dxr/build.py

```diff
@@ -212,7 +214,8 @@ def delete_index_quietly(es, index):
             # Substitute the format, tree name, and uuid into the index identifier.
             index = tree.es_index.format(format=FORMAT,
                                          tree=tree.name,
-                                         unique=uuid1())
+                                         unique=uuid1(),
+                                         config_path_hash=config.path_hash())
```

Comment:
We shouldn't need this for uniqueness, since "a random 14-bit sequence number" is part of uuid1() when called with no params. Is it here to help humans correlate an alias and its corresponding index?

Reply:
The problem with the UUID is that we don't know what UUID is going to be generated in advance and I don't have a good way to get the value actually generated here all the way to `_delete_es_indices`.  Guaranteeing that the index and the catalog both start with the same prefix and that this prefix depends only on the test being run make them easy to clean up with a wildcard.  If you can suggest a better way to deal with this, I'll be happy to adjust it.